app/addons/auth/components.react.jsx

```diff
@@ -239,9 +238,6 @@ define([
             <input id="password" type="password" name="password" placeholder= "Password" size="24"
               onChange={this.onChangePassword} />
             <p>
-              Non-admin users have read and write access to all databases, which
-              are controlled by validation functions. CouchDB can be configured to block all
-              access to anonymous users.
```

Comment:
Technically, to block anonymous users you need to fix admin party (aka create admin) and set [require_valid_user](http://docs.couchdb.org/en/1.6.1/config/auth.html#couch_httpd_auth/require_valid_user) as `true`, so everyone have to auth to send any request to CouchDB.